lib/fog/compute/google/models/firewall.rb

```diff
@@ -28,6 +28,15 @@ def save
           reload
         end
 
+        def update
```

Comment:
I like the idea of this, but dislike the idea of having every call to save having to first make a request to see if it should do an insert or save.  Is that what you are suggesting, or is there a different way you were envisioning?  Thanks!